### PR TITLE
SDK 1.9.1

### PR DESCRIPTION
**Pull request by Erol444**
WIP

### CODE REVIEW

**Comment by daniilpastukhov** [comment]
@Erol444 pushed changes from https://github.com/luxonis/depthai/pull/830 here

**Comment by daniilpastukhov** [comment]
I'll temporary return
```python
if self._config:
    nn_config = self._config.get("nn_config", {})
    if self._is_detector() and 'confidence_threshold' in nn_config:
        self.node.setConfidenceThreshold(float(nn_config['confidence_threshold']))
    meta = nn_config.get('NN_specific_metadata', None)
    if self._is_yolo() and meta:
        self.config_yolo_from_metadata(metadata=meta)
```
to the `NNComponent.__init__`.
It yields `AttributeError: 'NNComponent' object has no attribute 'node'` after it was moved to `NNComponent_parse_config`. cc @Erol444

**Comment by daniilpastukhov** [comment]
Also 1 suggestion: can we preferably use snake case over camel case in Python's API?
I think it's good from PEP8's and users' perspectives - all Python API is in snake case, while all exposed C++ API is in camel case (e.g., dai.Node methods are in camel case since they are implemented in C++, while dai_sdk.OakCamera methods are in snake case). Thoughts @Erol444?

**Comment by Erol444** [comment]
Yes sure @daniilpastukhov

**Comment by daniilpastukhov** [comment]
@Erol444 made last changes, tested on examples and experiments, everything LGTM. I think we can release 1.9.1 👍

File: depthai_sdk/src/depthai_sdk/nn_models/deeplabv3/model.yml [hunk 1]
```diff
@@ -7,7 +7,7 @@ files:
     size: 138536
     sha256: bd03684c902e99df3156d331167d94d965ad12193f2a8e26903be27ac17414fd
     source: https://blobconverter.s3.us-west-2.amazonaws.com/deeplabv3.xml
-  - name: FP16/deeplabv3p_person.bin
+  - name: FP16/deeplabv3p_person.xml
```
Comment (Erol444) [inline depthai_sdk/src/depthai_sdk/nn_models/deeplabv3/model.yml:10]: Is this correct?

Comment (Erol444) [reply depthai_sdk/src/depthai_sdk/nn_models/deeplabv3/model.yml:10]: Also we might want to just move the model to depthai-model-zoo and use blobconverter to download/convert it

File: depthai_sdk/src/depthai_sdk/oak_device.py [hunk 2]
```diff
@@ -21,21 +21,82 @@ def imageSensors(self) -> List[dai.CameraBoardSocket]:
     def info(self) -> dai.DeviceInfo:
         return self.device.getDeviceInfo()
 
-    def initCallbacks(self, pipeline: dai.Pipeline):
+    def stats_report(self) -> Dict[str, Any]:
```
Comment (Erol444) [inline depthai_sdk/src/depthai_sdk/oak_device.py:24]: This looks very robothub-ish function - can we move such functions into Hub SDK, which can just access device object from OakCamera?

**Review by Erol444 — COMMENTED** [review]
A few comments, otherwise LGTM

File: depthai_sdk/src/depthai_sdk/oak_device.py (hunk 2 above)
Comment (daniilpastukhov) [reply depthai_sdk/src/depthai_sdk/oak_device.py:24]: Yes, sure. I will move it.

File: depthai_sdk/src/depthai_sdk/nn_models/deeplabv3/model.yml (hunk 1 above)
Comment (daniilpastukhov) [reply depthai_sdk/src/depthai_sdk/nn_models/deeplabv3/model.yml:10]: It's already in depthai zoo, I've replaced it with depthai's model.

**Review by daniilpastukhov — APPROVED** [review]
LGTM👍

**Pull request merged**